_pages/install/smartEdit-setup-instructions-for-spartacus.md

```diff
@@ -14,78 +14,139 @@ feature:
 
 ## Prerequisites
 
-For Spartacus to work with SmartEdit, you need SAP Commerce Cloud 1905 (or newer) with `spartacussampledata` installed.
+For Spartacus to work with SmartEdit, you need SAP Commerce Cloud 1905 (or newer) with the `spartacussampledata` installed.
 
-## Configuring SmartEdit to work with a Spartacus storefront
+## Configuring SmartEdit to Work With a Spartacus Storefront
 
-1. Build your Angular app, adding Spartacus libraries as normal. Make sure it's working before continuing. For more information, see [Building the Spartacus Storefront from Libraries]({{ site.baseurl }}{% link _pages/install/building-the-spartacus-storefront-from-libraries.md %}).
+The SmartEdit feature library is introduced with version 3.2 of the Spartacus libraries, and as a result, the steps for configuring SmartEdit to work with Spartacus are different, depending on whether or not you are using the SmartEdit feature library.
 
-2. Copy the SmartEdit file `webApplicationInjector.js` to the folder `src` of your Angular app.
+### Configuring SmartEdit to work with Spartacus 3.2 or Newer
 
-   This file to copy can be found in your SAP Commerce Cloud installation, in the following folder:
+The following steps are for configuring SmartEdit to work using the SmartEdit feature library.
 
-   ```javascript
-   hybris/bin/modules/smartedit/smarteditaddon/acceleratoraddon/web/webroot/_ui/shared/common/js/webApplicationInjector.js
+1. Build your Angular app, adding Spartacus libraries as normal.
+
+   Make sure the app is working before continuing. For more information, see [Building the Spartacus Storefront from Libraries]({{ site.baseurl }}{% link _pages/install/building-the-spartacus-storefront-from-libraries.md %}).
+
+1. Install the SmartEdit feature library by running the following schematics command:
+
+   ```bash
+   ng add @spartacus/smartedit
+   ```
+
+   You can also install the SmartEdit library manually, by adding it to `package.json`, and then installing it by running `yarn update`, or using `npm` commands. If you install the SmartEdit library with this approach, then after installation, you also need to either copy the `webApplicationInjector.js` file from `node_modules/@spartacus/smartedit/assets` to your application's asset folder, or else add `node_modules/@spartacus/smartedit/assets` into the `assets` array in `angular.json`, as shown in the following example:
```

Comment:
For the first sentence "You can also install the SmartEdit library manually, by adding it to `package.json`, and then installing it by running `yarn update`, or using `npm` commands.". There are many ways to install a library. I think it is better we just say this: "You can also install the SmartEdit library manually. For details check https://angular.io/guide/using-libraries"